x-pack/plugins/ml/public/components/annotations/annotations_table/annotations_table.js

```diff
@@ -379,14 +389,39 @@ const AnnotationsTable = injectI18n(class AnnotationsTable extends Component {
       });
     }
 
+    const actions = [];
+
+    actions.push({
+      render: (annotation) => {
+        const editAnnotationsTooltipText = (
+          <FormattedMessage
+            id="xpack.ml.annotationsTable.editAnnotationsTooltip"
+            defaultMessage="Edit Annotation"
```

Comment:
Should use sentence case here `Edit annotation`

Reply:
Fixed in ec045d2492.